hooks/git/index.js

```diff
@@ -0,0 +1,34 @@
+const emoji = require('node-emoji')
+const chalk = require('chalk')
+const spawn = require('./../../utils/spawnHelper')
+
+const question = {
+  name: 'git',
+  type: 'confirm',
+  message: 'Would you like to initialize a git repository?'
+}
+
+
+function func (cwd, folderName) {
+
+  console.log('\n\n')
+  console.log(`${emoji.get('fire')}  ${chalk.cyan('Initializing git repository')} ${emoji.get('fire')}`)
+  console.log('\n\n')
+
+  const command = 'git init'
+  const terminalOpts = {
+    cwd: `${cwd}/${folderName}`,
+    shell: true,
+    stdio:'inherit',
+  }
+
+  return spawn(command, [], terminalOpts)
```

Comment:
So, We tried to find if has some issue on create-react-app related with that and I've found this PR https://github.com/facebookincubator/create-react-app/pull/1288

So, looks like that will be added at some moment, what do you think about test if already has a git started? maybe just test if exists .git folder.